### PR TITLE
Feat/hooks test

### PR DESCRIPTION
**Pull request by rajat693**

### CODE REVIEW

File: example/storybook-nativewind/src/components/Box/Box.tsx [hunk 1]
```diff
@@ -1,13 +1,26 @@
...
     <Box
-      className={`justify-center items-center bg-blue-500 w-[${props.w}px] h-[${props.h}px]`}
+      className={`justify-center items-center bg-blue-500 w-[${props.w}px] h-[${props.h}px] gap-4`}
+      // style={{
+      //   flexDirection: media.sm ? 'row' : 'column',
+      // }}
     >
       <Text className="font-bold text-typography-0">BOX</Text>
+      {/* <Box className="bg-red-300 text-center"> //testing hook
+        <Text className="font-bold text-typography-0">BOX 1</Text>
```
Comment (Viraj-10) [inline example/storybook-nativewind/src/components/Box/Box.tsx:18]: Remove this commented code.

File: example/storybook-nativewind/src/hooks/useBreakpointValue/index.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import { getTailwindConfigBreakpoints } from '@/utils/getTailwindConfigBreakpoints';
+
+const breakpoints = getTailwindConfigBreakpoints();
+
+export const useBreakpointValue = (values: any) => {
+  const { width } = useWindowDimensions();
+  const [value, setValue] = useState(values.base);
+
+  useEffect(() => {
+    if (width >= breakpoints['2xl'] && values['2xl'] !== undefined) {
```
Comment (Viraj-10) [inline example/storybook-nativewind/src/hooks/useBreakpointValue/index.ts:12]: remove hardcode break-points. 
check `@gluestack-ui/nativewind-utils/useBreakPointValue.ts`

File: example/storybook-nativewind/src/hooks/useMedia/index.ts [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    'base': false,
+    'sm': false,
+    'md': false,
+    'lg': false,
+    'xl': false,
+    '2xl': false,
+  });
+
+  useEffect(() => {
+    setMedia({
```
Comment (Viraj-10) [inline example/storybook-nativewind/src/hooks/useMedia/index.ts:19]: remove hardcode values of breakpoints.

**Review by Viraj-10 — CHANGES_REQUESTED** [review]

File: example/storybook-nativewind/src/hooks/useMedia/index.ts (hunk 3 above)
Comment (rajat693) [reply example/storybook-nativewind/src/hooks/useMedia/index.ts:19]: I haven't added any hardcode values those were coming bydefault from `tailwind`

**Pull request closed without merging**